anise/src/structure/planetocentric/ellipsoid.rs

```diff
@@ -58,6 +58,40 @@ impl Ellipsoid {
             polar_radius_km,
         }
     }
+
+    /// Builds new WGS84 Ellipsoid shape, used by all GPS vehicles.
+    /// <https://gisgeography.com/ellipsoid-oblate-spheroid-earth>
+    pub const fn gps_wgs84() -> Self {
+        const FLATTENING: f64 = 1.0 / 298.257223563_f64;
+        const SEMI_MAJOR_EQUATORIAL_RADIUS_KM: f64 = 6378.137_f64;
+        const SEMI_MINOR_EQUATORIAL_RADIUS_KM: f64 = 6356.7523142_f64;
+        // mean equatorial radius R = (M+m)/2
+        const R: f64 = (SEMI_MAJOR_EQUATORIAL_RADIUS_KM + SEMI_MINOR_EQUATORIAL_RADIUS_KM) / 2.0;
+        // flattening = (R - p)/R
+        const POLAR_RADIUS_KM: f64 = R - FLATTENING * R;
+        Self {
```

Comment:
By following the example above, you can define that ellipsoid in your code when you build the PCA file by using the public fields. ANISE will rebuild the map of the data in the file and when it's reloaded.